python/deep_slam/datasets/office.py

```diff
@@ -0,0 +1,30 @@
+import urllib.request
+from zipfile import ZipFile
+import os
+from pathlib import Path
+from deep_slam.datasets.VideoDataset import VideoDataset
+
+home = str(Path.home())
+data_dir = os.path.join(home, ".keras/datasets")
```

Comment:
you can just use `os.expanduser('~/.keras/datasets')`, I would also make data_dir argument of load_data to avoid having global variables